src/AlpineSkiHouse.Web/Models/Scan.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AlpineSkiHouse.Models
+{
+    public class Scan
+    {
+        public int Id { get; set; }
+
+        public int CardId { get; set; }
```

Comment:
I don't think there is anything about our model that would stop the case you described from working.